app/models/edition.rb

```diff
@@ -701,6 +701,10 @@ def public_url(options = {})
     website_root + public_path(options)
   end
 
+  def force_scheduled?
```

Comment:
The commit references `force_published?` but should be `force_scheduled?`

Reply:
Good spot! Thanks. Have sorted